### ISSUE TITLE
Bug: 

### ISSUE THREAD

**Issue by DangerousVanilla** [sloukit/pydew-valley-uzh#250]
#### Bug Description
When 'tutorial_status' is set to false in the save.json file, the game crashes when the player rapidly attempts to move in different directions in the startup sequence.

#### Expected Behavior
I believe character movement is supposed to just be disabled during the starting tutorial sequence.

#### Steps to Reproduce
1. Open the save.json file and change tutorial_status from 'true' to 'false'.
2. Launch the game and enter your sign in token
3. During the scripted sequence when you first see your character, spam the movement direction buttons and wait for an index out of range crash coming from src/gui/interface/dialog.py, line 182 in '_get_current_tb'

#### Additional Context
Here's my console output during the crash for reference:

Traceback (most recent call last):
  File "d:\GitHub\pydew-valley-uzh\main.py", line 413, in <module>
    asyncio.run(game.run())
  File "C:\Users\rayno\AppData\Roaming\uv\python\cpython-3.12.8-windows-x86_64-none\Lib\asyncio\runners.py", line 194, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\rayno\AppData\Roaming\uv\python\cpython-3.12.8-windows-x86_64-none\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\rayno\AppData\Roaming\uv\python\cpython-3.12.8-windows-x86_64-none\Lib\asyncio\base_events.py", line 686, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "d:\GitHub\pydew-valley-uzh\main.py", line 400, in run
    self.tutorial.update(is_game_paused)
  File "d:\GitHub\pydew-valley-uzh\src\tutorial\tutorial.py", line 241, in update
    self.check_tasks(game_paused)
  File "d:\GitHub\pydew-valley-uzh\src\tutorial\tutorial.py", line 123, in check_tasks
    and self.dialogue_manager._get_current_tb().finished_advancing
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\GitHub\pydew-valley-uzh\src\gui\interface\dialog.py", line 182, in _get_current_tb
    return self._tb_list[self._msg_index]
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
IndexError: list index out of range


### Version

0.1.0

### Which aspect(s) of the project is this bug affecting?

game-gameplay, game-playtesting

### Screenshots

_No response_

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

**Comment by DangerousVanilla** [comment]
Actually the attempt at controlling the character doesn't have to be quick apparently, you simply have to attempt to move in each direction (up, down, left, right) in any order and it crashes on the last direction you press as far as I can tell.

**Comment by DangerousVanilla** [comment]
Resolved in PR#251
